server/controllers/userAuth.js

```diff
@@ -448,6 +448,27 @@ const deleteAccount = async (req, resp) => {
   return resp.send();
 };
 
+const updateEmailNotificationSettings = async (req, resp) => {
+  const { never, weekly } = req.body;
+  const { _id } = req.user;
+
+  try {
+    await User.findOneAndUpdate(
+      { _id },
+      {
+        emailNotificationSettings: {
+          never,
+          weekly
```

Comment:
Maybe, like in list type case, we should use some enums and store:
```
settings: {
  notificationFrequency: 'weekly'
}
```
If switches would be changed to radio/select we will have one value then, which could be a string instead of  boolean